extensions/pgmq/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pgmq"
-version = "0.2.1"
+version = "0.3.0"
```

Comment:
were there new features added to the extension? I think there were new features to the crate but they have not been added to the extension, afaik.

Reply:
No, I guess not. What's the best way to find and add those features? I see there have been 4 bumps to the crate since the extension was last updated.

Reply:
If there's new new features added to the extension then this should be a patch bump to `0.2.2`

Reply:
I'm not sure what the _best_ way is but there were two feature releases to the crate; `send_delay` in 0.9 and `set_vt` in 0.10.

Reply:
Okay, thanks! I'll look into adding those.